Palindrome.java

```diff
@@ -25,6 +27,64 @@
  */
 public class Palindrome {
     public static void main(String[] args) {
-        // Your code goes here :)
+
+        Scanner input  = new Scanner(System.in);
+        byte palindromeSize;
+
+        System.out.printf("Please enter a number(int), between 0 and 1,000,000,000: ");
+        long inputNum = input.nextInt();
+
+        while (inputNum <= 0 || inputNum > 1000000000) {
+            System.out.printf("Please enter a valid number: ");
+            inputNum = input.nextLong();
+        }
+        input.close();
+
+        palindromeSize = checkIfPalindrome(inputNum);
+        if (palindromeSize == 0) { return; }
+
+        byte[] palindromeArr = new byte[palindromeSize];
+        enterPalindromeIntoArray(palindromeArr, inputNum);

```

Comment:
The method itself can return the newly created array instead of creating it and passing it over

Reply:
Correct. I'll revise.